storio-content-resolver/src/androidTest/java/com/pushtorefresh/storio/content_resolver/impl/ObserveChangesTest.java

```diff
@@ -0,0 +1,112 @@
+package com.pushtorefresh.storio.content_resolver.impl;
+
+import android.net.Uri;
+import android.support.annotation.NonNull;
+
+import com.pushtorefresh.storio.contentresolver.Changes;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+
+import rx.Observer;
+import rx.Subscription;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.schedulers.Schedulers;
+
+public class ObserveChangesTest extends BaseSubscriptionTest {
+
+    public class SubscribeStub extends BaseSubscribeStub<Changes> {
+
+        public SubscribeStub(@NonNull Queue<Changes> expected) {
+            super(expected);
+        }
+
+        @Override
+        public Subscription subscribe() {
+            return storIOContentResolver
+                    .observeChangesOfUri(Uri.parse(UserMeta.CONTENT_URI))
+                    .subscribeOn(Schedulers.io())
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .subscribe(new Observer<Changes>() {
```

Comment:
Here and everywhere below you can use overload of subscribe

```java
.subscribe(new Action1<ResultType> { });
```

Instead of monstrous `Observer` with three methods. `RxJava` will throw exception if `onError` won't be handled -> test will fail, `onComplete` is not required for your tests  :)